lib/galaxy/managers/markdown_parse.py

```diff
@@ -32,6 +32,7 @@
     "generate_galaxy_version": [],
     "generate_time": [],
     "invocation_time": ["invocation_id"],
+    "visualization": ["id", "history_dataset_id"],
```

Comment:
visualization_id I think.

Reply:
I don't think it's the case. it fetches `${getAppRoot()}api/plugins/${this.args.id}` at [this line](https://github.com/galaxyproject/galaxy/pull/10288/files#diff-e3dc9501b8af517e5d7201a3da7e64daR43) and http://localhost:8080/api/plugins/nvd3_bar_stacked request on network tab works as expected